kura/org.eclipse.kura.rest.identity.provider/OSGI-INF/IdentityRestServiceV2.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2024 Eurotech and/or its affiliates and others
+  
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+ 
+    SPDX-License-Identifier: EPL-2.0
+    
+    Contributors:
+     Eurotech
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" 
+    immediate="true" 
+    name="org.eclipse.kura.internal.rest.identity.provider.IdentityRestServiceV2">
+    
+    <implementation class="org.eclipse.kura.internal.rest.identity.provider.IdentityRestServiceV2" />
+
+    <reference interface="org.eclipse.kura.cloudconnection.request.RequestHandlerRegistry"
+               bind="bindRequestHandlerRegistry"
+               unbind="unbindRequestHandlerRegistry"
+               cardinality="0..n"
+               name="RequestHandlerRegistry"
+               policy="dynamic" />
+
+    <reference interface="org.osgi.service.useradmin.UserAdmin"
+               bind="bindUserAdmin"
+               cardinality="1..1"
+               name="UserAdmin"
+               policy="static" />
+
+    <reference interface="org.eclipse.kura.identity.PasswordStrengthVerificationService"
+               bind="bindPasswordStrengthVerificationService"
+               cardinality="1..1"
+               name="PasswordStrengthVerificationService"
+               policy="static" />
+               
+    <reference interface="org.eclipse.kura.identity.IdentityService" 
+               bind="bindIdentityService" 
+               cardinality="0..1"  
+               name="IdentityService" 
+               policy="static" />
+
+    <service>
+        <provide interface="org.eclipse.kura.internal.rest.identity.provider.IdentityRestServiceV2" />
```

Comment:
Should we export this as a service in the framework?

Reply:
it's not an issue since they are not api, similar services are exposed in others rest bundle